### PR TITLE
PAF-301: Amend date and time format

### PR DESCRIPTION
**Pull request by Rhodine-orleans-lindsay**
## What?
-  Amend field format - [PAF-301](https://collaboration.homeoffice.gov.uk/jira/browse/PAF-301)
## Why?
- 'Date received' field in IMS showing incorrect date on submission 
- time field had a typo and showed the time incorrectly when minutes was in single digits
## How?
- write time and date field for eform with capital letters to match ims fields in case of case sensitivity.
- set date format to en-GB ('dd/mm/yyyy)'
- add correct time field for eform
- add 0 in front of single digit minutes in time field
## Testing?
- all tests passing locally, tested on branch
## Anything Else? (optional)
- update yarn.lock to fix vulns
- added pr template
- added .DS_Store to .gitignore
-  removed commented out console.logs
## Check list
- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant) - not applicable
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

File: models/ims-model.js [hunk 1]
```diff
@@ -58,11 +58,12 @@ const setEformValue = (eform, fieldName, fieldValue) => {
...
   const today = new Date();
-  const time = today.getHours() + ':' + today.getMinutes();
+  //add 0 in front of single digit minutes
+  const time = today.getHours() + ':' + (today.getMinutes() < 10 ? '0' : '') + today.getMinutes();
 
   setEformValue(eform, 'caseid', caseRef);
-  setEformValue(eform, 'dtborec', today.toLocaleDateString());
-  setEformValue(eform, 'tmboec', time);
+  setEformValue(eform, 'dtBORec', today.toLocaleDateString("en-GB"));
+  setEformValue(eform, 'tmBORec', time);
```
Comment (tobyschaefferHO) [inline models/ims-model.js:66]: per https://collaboration.homeoffice.gov.uk/jira/browse/PAF-299 this looks right.

File: pull-request-template.md
Comment (tobyschaefferHO) [inline pull-request-template.md:1]: Shouldn't this be included in a separate PR as it's not related to the correction on the date and time format?

File: models/ims-model.js [hunk 2]
```diff
@@ -58,11 +58,12 @@ const setEformValue = (eform, fieldName, fieldValue) => {
 
 const setEformValues = (eform, caseRef) => {
   const today = new Date();
-  const time = today.getHours() + ':' + today.getMinutes();
+  //add 0 in front of single digit minutes
+  const time = today.getHours() + ':' + (today.getMinutes() < 10 ? '0' : '') + today.getMinutes();
```
Comment (tobyschaefferHO) [inline models/ims-model.js:62]: 👍

**Review by tobyschaefferHO — APPROVED** [review]
update to ims-model.js looks good.  
It seems like the PR template file should be in a separate PR, but approving approving as I'm new to the team!

File: pull-request-template.md
Comment (Rhodine-orleans-lindsay) [reply pull-request-template.md:1]: Thanks Toby, something like this is quite low priority so creating a separate ticket for it often gets relegated to the backlog for a while. I find it more time efficient to add it in as an 'anything else' when I notice it's missing, so it  is available for use as soon as possible.
